### PR TITLE
Prevent abbreviation from being overwritten in sub-agencies

### LINKED ISSUE #449: Allow searching by agency abbrevation
I would like to search for "CBP" or "CIS", because I don't know the full name of the agency (Customs and Border Protection).

### PR DESCRIPTION
**Pull request by ramirezg**
+ added tests and cleaned some duplicate tests. 

should fix #449

The load function is working, but I still think we need to break it up more for allow for more testing.

### CODE REVIEW

File: foia_hub/tests/test_loading.py [hunk 1]
```diff
@@ -214,3 +212,12 @@ def test_extract_non_tty_phone(self):
         public_liaison['phone'] = []
         phone = extract_non_tty_phone(public_liaison)
         self.assertEqual(None, phone)
+
+    def test_build_abbreviation(self):
+        """ Test that abbreviations are build correctly """
```
Comment (khandelwal) [inline foia_hub/tests/test_loading.py]: s/build/built

**Comment by khandelwal** [comment]
Every time we touch the loading code, we can work towards breaking it down and making it more testable. However, if I read coveralls right, the loading code has decent test coverage.

**Comment by khandelwal** [comment]
There's an extremely minor documentation thing that I pointed out. Otherwise this is good to go.

**Comment by ramirezg** [comment]
@khandelwal, okay the docs should be updated.

**Pull request merged**